src/java.desktop/share/classes/com/sun/beans/introspect/MethodInfo.java

```diff
@@ -80,12 +97,18 @@ static List<Method> get(Class<?> type) {
                             // method = null; // ignore inaccessible methods
                         }
                     }
-                    if (method != null) {
-                        if (list == null) {
-                            list = new ArrayList<>();
-                        }
-                        list.add(method);
-                    }
+                    if (method != null)
+                        (list = createIfNeeded(list)).add(method);
+                }
+            }
+
+            // Add default methods inherited from interfaces
+            for (Class<?> iface : type.getInterfaces()) {
+                if (IGNORABLE_INTERFACES.contains(iface))
+                    continue;
+                for (Method method : iface.getMethods()) {
+                    if ((method.getModifiers() & Modifier.ABSTRACT) == 0)
+                        (list = createIfNeeded(list)).add(method);
                 }
```

Comment:
Please always use braces even if the body has only one statement.

Reply:
OK, I'll fix. I've seen examples of both styles in the JDK so am never really sure.

Reply:
In client libs, we tend to use the braces all the time.

By this comment, I meant adding braces to all the statements where you omitted them, including the test.

If you look through `MethodInfo.java` class, you see that braces are used consistently even if the body has only one statement.

Reply:
Fixed in ac90a10e238.